plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergColumnHandle.java

```diff
@@ -77,6 +82,41 @@ public String getName()
         return columnIdentity.getName();
     }
 
+    @JsonProperty
+    public List<String> getPath()
+    {
+        return path;
+    }
+
+    public List<String> getFullPath()
+    {
+        return ImmutableList.<String>builder()
+                .addAll(getPath())
+                .add(getName())
+                .build();
+    }
+
+    public String getQualifiedName()
+    {
+        if (path.isEmpty()) {
+            return getName();
+        }
+
+        return String.join(".", path) + "." + getName();
+    }
+
+    public boolean isBaseColumn()
+    {
+        return path.isEmpty();
+    }
+
+    public boolean isParentOf(IcebergColumnHandle otherColumn)
+    {
+        List<String> myPath = getFullPath();
+        List<String> otherPath = otherColumn.getFullPath();
+        return otherPath.size() >= myPath.size() && otherPath.subList(0, myPath.size()).equals(myPath);
```

Comment:
`>=` or `>`? Is `x` parent of `x`

Reply:
Hmm, for the one place this is used it is important that x is a parent of x. Could probably inline this method though, I thought it was going to be more re-usable than it was.